src/shared/components/common/markdown-textarea.tsx

```diff
@@ -386,8 +386,9 @@ export class MarkdownTextArea extends Component<
           let textarea: any = document.getElementById(i.id);
           autosize.update(textarea);
           pictrsDeleteToast(
-            i18n.t("click_to_delete_picture"),
-            i18n.t("picture_deleted"),
+            i18n.t("click_to_delete_picture").concat(' (', file.name,')'),
```

Comment:
Use string literals, not concat. I'd prefer these to be colons as well, so:

```
`${i18n.t("click_to_delete_picture)}: ${file.name}`
```

Same for the other two.

Reply:
Alright, we need to update translations then. I'll do a pull request.

Reply:
I did this change in the asyn image upload branch now though.